console/console-init/ui/src/graphql-module/queries/iot_project.ts

```diff
@@ -104,8 +104,7 @@ const RETURN_IOT_PROJECTS = (filterObj?: any, queryResolver?: string) => {
 
   if (filterObj) {
     //TODO
-    filter = "`$.kind`='IoTProject'";
-    // filter = FILTER_RETURN_IOT_PROJECTS(filterObj);
+    FILTER_RETURN_IOT_PROJECTS(filterObj);
```

Comment:
missing to assign filter variable.
filter=FILTER_RETURN_IOT_PROJECTS(filterObj);